score-lang-cli/src/test/java/org/openscore/lang/cli/SlangCLITest.java

```diff
@@ -45,9 +45,9 @@
 public class SlangCLITest {
 
     private final static String[] CONTEXT_PATH = { "classpath*:/META-INF/spring/test-spring-shell-plugin.xml" };
-    private final static String FLOW_PATH_SLAH = "C:\\\\flow.yaml";
+    private final static String FLOW_PATH_BACKSLASH_INPUT = "C:\\\\flow.yaml";
```

Comment:
the variable name was not consistent since the value contained backslash not forwardslash